sass/components/_editor.scss

```diff
@@ -0,0 +1,17 @@
+.chamel-color-picker {
+	position: absolute;
+	margin-top: 55px;
+	display: none;
+}
+
+.chamel-color-picker {
+	position: absolute;
+	margin-top: 55px;
+	display: none;
+}
+
+.chamel-color-picker-close {
```

Comment:
Should this be a subclass of the .chamel-color-picker.

.chamel-color-picker {
   // your styles here
   .chamel-color-picker-close {
      // closed subclass
   }
}